tests/test_subgraph.py

```diff
@@ -79,11 +81,22 @@ def test_get_balancer_pool_snapshots(chain, subgraph_all_chains, pool_snapshot_b
 
 
 @pytest.mark.parametrize("have_thegraph_key", [True, False])
-@pytest.mark.parametrize("subgraph_type", ['core', 'gauges', 'blocks', 'aura'])
+@pytest.mark.parametrize("subgraph_type", ["core", "gauges", "blocks", "aura"])
 def test_find_all_subgraph_urls(subgraph_all_chains, have_thegraph_key, subgraph_type):
-    if subgraph_all_chains.chain == 'sepolia' and subgraph_type in ['aura', 'blocks']:
-        pytest.skip(f'No {subgraph_type} subgraph exists on Sepolia')
-    os.environ['GRAPH_API_KEY'] = os.getenv('GRAPH_API_KEY') if have_thegraph_key else ""
+    if subgraph_all_chains.chain in ["sepolia", "mode"] and subgraph_type in [
+        "aura",
+        "blocks",
+    ]:
+        pytest.skip(
+            f"No {subgraph_type} subgraph exists on {subgraph_all_chains.chain}"
+        )
+    if subgraph_all_chains.chain == "fraxtal" and subgraph_type == "blocks":
```

Comment:
@gosuto-inzasheru is this a "hack" because fraxtal fee sweeping is not enabled? Other than that fraxtal is fully enabled with veBAL gauges etc.

Reply:
fraxtal simply doesnt have a blocks subgraph, so we have to skip that particular combination